config/initializers/bootstrap_errors.rb

```diff
@@ -0,0 +1,30 @@
+ActionView::Base.field_error_proc = Proc.new do |html_tag, instance|
+  html = %Q(<span class="field_with_errors">#{html_tag}</span>).html_safe
+
+  form_fields = [
+    'textarea',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.